src/components/AddressResolver/EndpointEntry/EndpointEntry.tsx

```diff
@@ -135,92 +138,131 @@ export const EndpointEntry: FunctionComponent<EndpointEntryProps> = ({
+              value={endpointApi}
+              onChange={onEndpointApiChanged}
+              errorMessage={inputErrorMessageEndpoint}
+            />
+          ) : (
+            <>{api}</>
+          )}
+        </td>
+        <td className="text-cloud-900 text-xl font-bold px-3 pt-3 md:hidden">API Header</td>
+        <td className="w-auto md:w-52">
+          {isEditable ? (
+            <Input
+              className="w-full"
+              placeholder="API Header"
+              value={endpointApiHeader}
+              onChange={onEndpointApiHeaderChanged}
+              errorMessage={inputErrorMessageApiHeader}
+            />
+          ) : (
+            <>{apiHeader}</>
+          )}
+        </td>
+        <td className="text-cloud-900 text-xl font-bold px-3 pt-3 md:hidden">API Key</td>
+        <td className="w-auto md:w-52" colSpan={isEditable ? 2 : 0}>
+          {isEditable ? (
+            <Input
+              className="w-full"
+              placeholder="API Key"
+              value={endpointApiKey}
+              onChange={onEndpointApiKeyChanged}
+              errorMessage={inputErrorMessageApiKey}
+            />
+          ) : (
+            <>{apiKey}</>
+          )}
+        </td>
+        {!isEditable && (
+          <td className={"w-28"}>
+            <div className="flex w-full ml-56 -mt-86 md:m-auto">
+              <Trash2 className="text-cerulean-200 cursor-pointer" onClick={removeEndpoint} data-testid="trash2-icon" />
               <Edit
+                className="text-cerulean-200 ml-3.5 cursor-pointer"
                 onClick={() => {
                   setEditable(true);
                 }}
                 data-testid="edit-icon"
               />
-            )}
-            <Trash2 onClick={removeEndpoint} data-testid="trash2-icon" />
-          </div>
-        </td>
+            </div>
+          </td>
+        )}
+      </tr>
+      {isEditable && (
+        <tr className={`${className} ${orderNumber % 2 !== 0 ? "md:bg-cerulean-50" : "bg-white"}`}>
+          <th className="hidden md:table-cell" />
+          <td colSpan={10}>
+            <div className="flex flex-row text-white text-base justify-center -mt-14 md:m-0">
+              <div
+                className="flex bg-rose rounded-xl py-2 px-2.5 w-20 h-9 justify-center items-center cursor-pointer"
+                onClick={removeEndpoint}
+              >
+                <h5>Delete</h5>
+              </div>
+              {isLoading ? (
+                <LoaderSpinner className="ml-10" />
+              ) : (
+                <div
+                  className="flex bg-cerulean rounded-xl py-2 px-2.5 w-20 h-9 ml-10 justify-center items-center cursor-pointer"
+                  onClick={onSave}
+                  data-testid="save-icon"
+                >
+                  <h5>Save</h5>
+                </div>
+              )}
+            </div>
+          </td>
+          <th className="hidden md:table-cell" />
```

Comment:
need to rework, table giving problems

Reply:
table done just left with the toggling row position (up down arrow) part for EndpointEntry